src/plugins/intel_gpu/src/graph/include/reshape_inst.h

```diff
@@ -28,6 +29,35 @@ struct typed_program_node<reshape> : public typed_program_node_base<reshape> {
 
     program_node& input() const { return get_dependency(0); }
 
+    bool is_runtime_propagatable_padding() const {
+        auto prim = typed_desc();
+        if (prim->mode == reshape::reshape_mode::squeeze || prim->mode == reshape::reshape_mode::unsqueeze)
+            return true;
+
+        // TODO: This function is to limit condition to a specific case (crop + reshape) among cases for the base mode
+        if (!input().is_type<crop>())
+            return false;
+
+        auto axis = input().as<crop>().get_primitive()->axis;
+        auto input_rank = input().get_output_layout(false).get_partial_shape().size();
+        auto input_last_dim = static_cast<int64_t>(input_rank - 1);
+        if (axis != input_last_dim)
+            return false;
+
+        auto output_pattern = prim->output_pattern;
```

Comment:
Is there any condition that the reshape's pattern input is constant? 
I am wondering about the case when reshape's pattern input is normal primitive (e.g., concat)

Reply:
Ah.. it is allowing only when the output_pattern is not empty below..

Reply:
How about checking output shape of reshape instead of pattern? Maybe some dimension is already static from the IR.

Reply:
update to check output partial shape of reshape instead of output pattern